apis/services/v1alpha1/auth_types.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/opendatahub-io/opendatahub-operator/v2/apis/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	AuthServiceName  = "auth"
+	AuthInstanceName = "auth"
```

Comment:
nit: we have been using the naming format `default-*`(e.g default-dsc, default-dsci, default-monitoring) , can we update the name here as well?

Reply:
I initially used that naming of default-auth, but @andrewballantyne made the point that it seems strange to use default-auth when it's intended to be a singleton and I agree. There will only ever be one, the system doesn't allow two. A prefix of default seemed strange. It implies that there might be other non-default auth instances or that we are creating a default because the user didn't provide an auth instance.

Reply:
@VaishnaviHire wonder if the same case should be made against components ?

Reply:
Yeah Andrew's and Steven's points make sense. I do not remember the reason why we went with `default-` prefix.  I am okay with removing the `default`, but we will have to have upgrade logic to update the names for components and dsc/dsci in ODH releases

Reply:
i guess:  we have default-dsc/default-dsci, we carried this "default-" to the component CRs.
and it is easier to have the pattern as filter.